src/bindings/python/src/openvino/utils/postponed_constant.py

```diff
@@ -7,28 +7,23 @@
 import openvino
 
 """Postponed Constant is a way to materialize a big constant only when it is going to be serialized to IR and then immediately dispose."""
-
+class PostponedConstant(openvino.Op):
+    def __init__(self, element_type: openvino.Type, shape: openvino.Shape, maker: Callable[[], openvino.Tensor]) -> None:
```

Comment:
When an operation is created in the op factory, will it be called with an empty list of arguments? @akuporos, please clarify if the automatically generated cloning will work for this kind of op.